src/backend/looped/self/persist_outfit.cpp

```diff
@@ -56,7 +56,14 @@ namespace big
 			folder saved_outfit_path = g_file_manager.get_project_folder("saved_outfits");
 			std::ifstream i(saved_outfit_path.get_file(persisting_outfit).get_path());
 			outfit.clear();
-			i >> outfit;
+			try
+			{
+				i >> outfit;
+			}
+			catch (const std::exception& e)
+			{
+				outfit = {};
```

Comment:
LOG(INFO) the exception string, and also zero out g.self.persist_outfit, otherwise it's just going to attempt to try again next frame.